cni/pkg/repair/netns.go

```diff
@@ -88,12 +82,11 @@ func getLinkWithDestinationOf(ip string) (netlink.Link, error) {
 //
 // Instead, we traverse the procfs. Comments on this method are inline.
 func getPodNetNs(pod *corev1.Pod) (string, error) {
-	// First, find the network namespace id by looking the interface with the given Pod IP.
-	// This could break on some platforms if they do not have an interface-per-pod.
-	wantID, err := findNetworkIDByIP(pod.Status.PodIP)
-	if err != nil {
-		return "", fmt.Errorf("network id: %v", err)
+	parsedPodAddr := net.ParseIP(pod.Status.PodIP)
```

Comment:
IIRC `net.ParseIP` doesn't handle 4in6 correctly: https://github.com/golang/go/issues/50844#issuecomment-1023441825

but it probably doesn't matter here